_posts/2017/2017-08-08-why-you-should-spend-time-on-your-translation-keys.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: Why you should spend time on your translation keys
+date: "2017-08-08 09:34:02"
+image: blog-header/translation-keys.jpg
+categories: tech-stories
+authors: ["Christina"]
+---
+Translation keys are not important at all?
+Spending time on them is a waste of time?
+Let me convince you otherwise.
+
+We just finished the revision of our translation keys because we noticed the value of a good key structure.
+In the following, I will tell you why we changed the old system and what have been the most important aspects during this task.
+
+## Structure
+
+We already had functional keys before.
+Everything worked out.
+But nobody really paid attention to an overall structure or any consistency.
```

Comment:
As further in the post you give an example of the new structure, maybe it would be good to give an example of the old structure?